components/Conversations/Detail/ConversationDetail.tsx

```diff
@@ -63,23 +64,33 @@ const useTranscriptionTimer = (isAudioOn: boolean, micActivity: number, saveCurr
   return isTranscribing
 }
 
-const ConversationDetail: React.FC<ConversationDetailProps> = ({ conversation }) => {
-  const { micActivity, isAudioOn, saveCurrentConversation } = useConversations()
+const ConversationDetail: React.FC<ConversationDetailProps> = ({ 
+  conversation 
+}) => {
+  const { micActivity, isAudioOn, saveCurrentConversation, conversations } = useConversations()
   const isTranscribing = useTranscriptionTimer(isAudioOn, micActivity, saveCurrentConversation)
 
+  // First priority: Show ActiveConversation when transcribing
+  if (isAudioOn && isTranscribing) {
+    return <ActiveConversation />
+  }
```

Comment:
@Jungyoonlim this is still preventing me from accessing any older conversations while there's an active conversation. please make sure you test this - keeps coming up in different PRs. let me know if you have any questions about this, we can talk through the logic